packages/block-editor/src/components/off-canvas-editor/leaf-more-menu.js

```diff
@@ -19,11 +19,69 @@ const POPOVER_PROPS = {
 	variant: 'toolbar',
 };
 
+const BLOCKS_THAT_CAN_BE_CONVERTED_TO_SUBMENU = [
+	'core/navigation-link',
+	'core/navigation-submenu',
+];
+
+function AddSubmenuItem( { block, clientId, onClose } ) {
+	const { insertBlock, replaceBlock, replaceInnerBlocks } =
+		useDispatch( blockEditorStore );
+
+	if (
+		BLOCKS_THAT_CAN_BE_CONVERTED_TO_SUBMENU.indexOf( block.name ) === -1
+	) {
+		return null;
+	}
```

Comment:
Question: for a11y reasons should the option remain but be disabled?

Reply:
good idea, done.